src/cli/README.md

```diff
@@ -2341,6 +2341,17 @@ Gets the IPv4 configured CIDR in the NAT64 translator.
 Done
 ```
 
+### nat64 cidr \<IPv4 address\>
```

Comment:
It might be useful to add the caveats noted in the API used for this command

https://github.com/openthread/openthread/blob/6ff277841ebd6045be87d8407c840df9bc35709c/include/openthread/nat64.h#L318

Mainly that it must have a non-zero prefix length and that the NAT64 translator will be reset and all existing sessions expired when this is used.

Reply:
I have added these caveats as a Note.